CONTRIBUTING.md

```diff
@@ -2,48 +2,49 @@
 
 <!-- TOC GFM -->
 
-- [Guidelines for contributing](#guidelines-for-contributing)
-- [Getting started](#getting-started)
-  - [Using multiple versions of Node](#using-multiple-versions-of-node)
-  - [Quick edits](#quick-edits)
-  - [Cloning vs Forking](#cloning-vs-forking)
-  - [Submitting a PR from a forked repo](#submitting-a-pr-from-a-forked-repo)
-  - [Submitting a PR from a cloned repo](#submitting-a-pr-from-a-cloned-repo)
-  - [Using the `develop` branch](#using-the-develop-branch)
-  - [Using Conventional Commits](#using-conventional-commits)
-    - [Use `chore`](#use-chore)
-    - [Use `fix`](#use-fix)
-    - [Use `feat`](#use-feat)
-  - [Draft PRs](#draft-prs)
-  - [Deploy previews with Amplify](#deploy-previews-with-amplify)
-- [Style guide adherence](#style-guide-adherence)
-- [Reusable components](#reusable-components)
-- [Technical reference contribution guidelines](#technical-reference-contribution-guidelines)
-- [Editing existing pages](#editing-existing-pages)
-- [Creating new pages](#creating-new-pages)
-- [Deleting pages](#deleting-pages)
-- [New Relic guides](#new-relic-guides)
-  - [Purpose](#purpose)
-- [Tips for writing guides](#tips-for-writing-guides)
-  - [Check for existing content](#check-for-existing-content)
-  - [Editing existing guides](#editing-existing-guides)
-  - [Creating new guides](#creating-new-guides)
-  - [Deleting guides](#deleting-guides)
-- [Related Pages](#related-pages)
-- [Updating navigation](#updating-navigation)
-  - [Example navigation change](#example-navigation-change)
-    - [EXISTING](#existing)
-    - [UPDATED](#updated)
-- [Split testing and running experiments](#split-testing-and-running-experiments)
-  - [Step 1](#step-1)
-  - [Step 2](#step-2)
-  - [Step 3](#step-3)
-  - [Step 4](#step-4)
-  - [Step 5](#step-5)
-- [Updating the documentation bundle](#updating-the-documentation-bundle)
-  - [Step 1: Update the release number in `gatsby-config`](#step-1-update-the-release-number-in-gatsby-config)
-  - [Step 2: Add any new APIs or components to our constants list](#step-2-add-any-new-apis-or-components-to-our-constants-list)
-  - [Step 3: Add any new APIs or components to the navigation](#step-3-add-any-new-apis-or-components-to-the-navigation)
+- [Table of Contents](#table-of-contents)
```

Comment:
This is another instance where it looks like your plugin is adding a bunch of nesting. 🤷‍♂️

Reply:
Yeah... it's very opinionated.. 🤷‍♂️